src/component/Header.css

```diff
@@ -1,5 +1,5 @@
 .bg-mskcc-header {
```

Comment:
The changes in this file is already included in https://github.com/genome-nexus/genome-nexus-frontend/pull/3, right? maybe remove them from this PR?